entrypoint.sh

```diff
@@ -1,46 +1,49 @@
 #!/bin/bash
+set -e
 
 # Set path
-echo '::group::Copying file from $WORKPATH to /tmp/gh-action'
+echo "::group::Copying file from $WORKPATH to /tmp/gh-action"
 WORKPATH=$GITHUB_WORKSPACE/$INPUT_PATH
 # Set path permision
 sudo -u builder mkdir /tmp/gh-action
-sudo -u builder cp -rfv $WORKPATH/.git /tmp/gh-action/.git
-sudo -u builder cp -fv $WORKPATH/PKGBUILD /tmp/gh-action/PKGBUILD
-cd /tmp/gh-action
-echo '::endgroup::'
+sudo -u builder cp -rfv "$WORKPATH"/.git /tmp/gh-action/.git
+sudo -u builder cp -fv "$WORKPATH"/PKGBUILD /tmp/gh-action/PKGBUILD
+cd /tmp/gh-action || exit
+echo "::endgroup::"
 
 # Update checksums
-echo '::group::Updating checksums on PKGBUILD'
 if [[ $INPUT_UPDPKGSUMS == true ]]; then
+    echo "::group::Updating checksums on PKGBUILD"
     sudo -u builder updpkgsums
     git diff PKGBUILD
+    echo "::endgroup::"
 fi
-echo '::endgroup::'
 
 # Generate .SRCINFO
-echo '::group::Generating new .SRCINFO based on PKGBUILD'
 if [[ $INPUT_SRCINFO == true ]]; then
+    echo "::group::Generating new .SRCINFO based on PKGBUILD"
     sudo -u builder makepkg --printsrcinfo > .SRCINFO
     git diff .SRCINFO
+    echo "::endgroup::"
 fi
-echo '::endgroup::'
 
 # Validate with namcap
-echo '::group::Validating PKGBUILD with namcap'
 if [[ $INPUT_NAMCAP == true ]]; then
-    namcap -i PKGBUILD
+    echo "::group::Validating PKGBUILD with namcap"
+    namcap -i PKGBUILD || exit
```

Comment:
Shouldn't the action fail, if package validation fails?
Currently it will bail out silently.
I would remove the ` || exit`. Because of the `set -e` flag, the script will abort if `namcap` exits with exit code != 0.

Reply:
Done. Both `|| exit` already removed.